### DISCUSSION TITLE
Incorrect Google page simulation

### DISCUSSION THREAD

**Discussion by casaldisenio** [responsively-org/responsively-app#996] [Show and tell]
I wanted to know if the app really shows how a site would look like on different devices since it shows the Google page incorrectly (see attached image). Thank you very much 
![screenResponsivelyApp](https://github.com/responsively-org/responsively-app/assets/5350361/11cfd043-b126-47c3-a735-22d58451c78a)

**Comment by manojVivek** [comment]
@casaldisenio There was a bug in the device user-agents which we [resolved](#990) in the latest release, v1.5.0. 
Please update the app to the latest version and then try again. 

Here's the screenshot in the latest version:
<img width="2314" alt="Screenshot 2023-06-26 at 8 54 21 AM" src="https://github.com/responsively-org/responsively-app/assets/1283424/91ddbdf4-9976-4b1e-a5e8-a4f81477ad9b">

**Reply by casaldisenio** [reply]
perfect! thank you so much!
